templates/include/glean/event.1.schema.json

```diff
@@ -1,17 +1,17 @@
 {
-  "type": "array",
-  "items": [
-    {
-      "type": "integer",
-      "minimum": 0
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "timestamp": {
+      "type": "integer"
```

Comment:
We seem to have lost the `"minimum": 0` on `timestamp`

Reply:
Done!